container/container_unix.go

```diff
@@ -523,7 +524,7 @@ func (container *Container) UnmountIpcMounts(unmount func(pth string) error) {
 			warnings = append(warnings, err.Error())
 		} else if shmPath != "" {
 			if err := unmount(shmPath); err != nil {
-				warnings = append(warnings, fmt.Sprintf("failed to umount %s: %v", shmPath, err))
+				warnings = append(warnings, fmt.Sprintf("failed to unmount %s: %v", shmPath, err))
```

Comment:
I wonder if this was on purpose (and referring to [`umount`](http://man7.org/linux/man-pages/man8/umount.8.html)), either way, I think it's fine